### PR TITLE
Navbar bugfix

### PR DESCRIPTION
**Pull request by kfang**
- navigation bar should be gone
- will use separate branch to refactor away old code
- navigation replaced with simple red menu box

Other notes:
- not sure where to put it for the game, waiting on the viewport merge to figure this out
- LMK if you see any bugs, tests all pass locally

### CODE REVIEW

File: the_game_bazaar/static/the_game_bazaar/js/home.js [hunk 1]
```diff
@@ -42,17 +42,23 @@ $().ready(function(){
...
+    } else {
+        change_page(templates, 'title', false, false);
+    }
+
 });
 
-function change_page(templates, page, force){
+function change_page(templates, page, force, change_hash){
     if (force){
         force = force;
```
Comment (jamesmuerle) [inline the_game_bazaar/static/the_game_bazaar/js/home.js:62]: huh? Why is this line here?

Comment (kfang) [inline the_game_bazaar/static/the_game_bazaar/js/home.js:62]: its the change page function
-templates is the template dictionary
-page is the key into the dictionary
-force is to make the make the page change even if it is already on that page (a.k.a refresh)
-change_hash is whether or not to change the hash in the url bar.  true means to change it, and false means don't change it.

Comment (hellochar) [inline the_game_bazaar/static/the_game_bazaar/js/home.js:62]: add that documentation you just wrote to the code. also you can create 'default' values for `force` and `change_hash` by doing `force = force || true;` so you don't have to explicitly specify the same values all over the codebase

File: lib/templates/lib/navbar.html [hunk 2]
```diff
@@ -1,38 +1,75 @@
...
+
+        $('#menu-system #button').click(function(){
+            $('#menu-system #dropdown').slideToggle();
+        });
+
+    })
+</script>
+
+<style>
+    #menu-system {
```
Comment (hellochar) [inline lib/templates/lib/navbar.html:14]: i was under the impression inline styles weren't good for maintainability

File: the_game_bazaar/static/the_game_bazaar/js/home.js [hunk 3]
```diff
@@ -42,17 +42,23 @@ $().ready(function(){
...
-    change_page(templates, 'title');
-
     //mostly just hides all the divs
     initialize();
 
     //adds click functions
     bind_divs();
+
+    //look at the anchor and change page accordingly
+    var hash = window.location.hash.replace('#', '');
```
Comment (hellochar) [inline the_game_bazaar/static/the_game_bazaar/js/home.js:52]: I'd do `window.location.hash.substring(1)`, unless there's a reason you're actually replacing all `#`'s

**Comment by hellochar** [comment]
looks good! Is the menu supposed to be hidden in the game client when people actually start playing? maybe bring it up with F10 or something (if we can capture the f-keys that is)

File: the_game_bazaar/static/the_game_bazaar/js/home.js (hunk 1 above)
Comment (kfang) [inline the_game_bazaar/static/the_game_bazaar/js/home.js:62]: if you don't define it, doesn't if(force) just default to false? Also, the last two should be optional, they're everywhere cuz i did a find and replace all :P

File: lib/templates/lib/navbar.html (hunk 2 above)
Comment (kfang) [inline lib/templates/lib/navbar.html:14]: Its not, but we have them all over the place too... I said refactoring will happen in another branch.

File: the_game_bazaar/static/the_game_bazaar/js/home.js (hunk 3 above)
Comment (kfang) [inline the_game_bazaar/static/the_game_bazaar/js/home.js:52]: didn't think of that...

**Comment by kfang** [comment]
I didn't want to futz with the game.html since the other requests were working with it and I didn't want git to complain.  Will add in when we get together.

**Comment by hellochar** [comment]
sure, i'm good with merging then
